apps/codelab/src/app/components/breadcrumb/breadcrumb.component.html

```diff
@@ -0,0 +1,8 @@
+<h1>
+  <a routerLink="/">Angular Codelab</a>
+  {{separator}}
+  <!--TODO(kirjs): Make milestone a dropdown-->
+  {{milestone}}
```

Comment:
Don't you wanna add link to the start milestone page?

Something like:
![image](https://user-images.githubusercontent.com/40883170/52531404-0fe23180-2d26-11e9-9f9a-9642c92de7dd.png)

![image](https://user-images.githubusercontent.com/40883170/52531409-2092a780-2d26-11e9-825c-885662204363.png)

Reply:
Would there be much value? 
I think it would be cool to have dropdowns allowing to do to any milestone or any slide here, but I couldn't imagine a good use case for going to the first slide